README.md

```diff
@@ -0,0 +1,23 @@
+nereid-activity-stream
+======================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-activity-stream.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-activity-stream)
+[![Downloads](https://pypip.in/download/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Development Status](https://pypip.in/status/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Coverage Status](https://coveralls.io/repos/openlabs/nereid-activity-stream/badge.svg?branch=develop)](https://coveralls.io/r/openlabs/nereid-activity-stream?branch=develop)
+
+
+| Authors        | [Openlabs Technologies & Consulting (P) Limited][]          
```

Comment:
While this gives the resultant table, it doesn't look good in markdown.

Try to beautify it a bit.